src/pages/PokemonDetailsPage/PokemonDetailsPage.jsx

```diff
@@ -0,0 +1,152 @@
+          species: response.data.species.name,
+          frontImageUrl: response.data.sprites.front_default,
+          backImageUrl: response.data.sprites.back_default,
+          hp: response.data.stats[0].base_stat,
+          attack: response.data.stats[1].base_stat,
+          defense: response.data.stats[2].base_stat,
+          type: response.data.types[0].type.name,
+          weight: weightInKg,
+          height: heightInMetres,
+        });
+      })
+      .catch((error) => {
+        if (error.response) {
+          if (error.response.status === 404) {
+            navigate("/404");
+            return;
+          }
+        }
+      })
+      .finally(() => {
+        setShowBackImage(false);
+      });
+  };
+
+  const copyToClipbaord = () => {
+    navigator.clipboard.writeText(window.location.href);
+  };
+
+  const handleImageBoolean = () => {
+    if (showBackImage === true) {
+      setShowBackImage(false);
+    } else {
+      setShowBackImage(true);
+    }
+  };
+
+  return (
+    <div className="display-section">
+      <div className="snackbar-area">
+        <Snackbar message="URL copied to clipboard!" ref={snackbarRef} />
+      </div>
+      <div className="stats-window">
+        <div className="button-and-title">
+          <button
+            className="copy-to-clipboard"
+            onClick={() => {
+              snackbarRef.current.show();
+              copyToClipbaord();
+            }}
+          >
+            ⎘
+          </button>
+          <h1 className="stats-title">{pokemonInfo.name}</h1>
+        </div>
+        <div className="text-and-image-wrapper">
+          <div className="text-stats">
+            <p>
+              <span className="bold">Species:</span> {pokemonInfo.species}
+            </p>
+            <p>
+              <span className="bold">Type:</span> {pokemonInfo.type}
+            </p>
+            <p>
+              <span className="bold">HP:</span> {pokemonInfo.hp}
+            </p>
+            <p>
+              <span className="bold">Attack:</span> {pokemonInfo.attack}
+            </p>
+            <p>
+              <span className="bold">Defence:</span> {pokemonInfo.defense}
+            </p>
+            <p>
+              <span className="bold">Height:</span> {pokemonInfo.height}
+              {" (m)"}
+            </p>
+            <p>
+              <span className="bold">Weight:</span> {pokemonInfo.weight}
+              {" (kg)"}
+            </p>
+          </div>
+          <div className="image-window">
+            {showBackImage ? (
+              <img
+                className="pokemon-image"
+                src={pokemonInfo.backImageUrl}
+                alt="front view of chosen pokemon"
+              ></img>
+            ) : (
+              <img
+                className="pokemon-image"
+                src={pokemonInfo.frontImageUrl}
+                alt="front view of chosen pokemon"
+              ></img>
+            )}
```

Comment:
I would look into how to do this with CSS!

You can do some stuff with CSS backface visiblity and `transform: rotate` that means you don't need JS here to manage the image that can be seen. 

Happy to chat in a call about it!

Reply:
So I've had a go with this. I've got it so that it functions identically before, now using css classes instead of conditionally rendering with Javascript. However, I haven't implemented any kind of animation or backface-visibility, I couldn't quite work out that part.